src/backend/web/graphql/index.js

```diff
@@ -61,6 +62,7 @@ module.exports.typeDefs = graphql`
     id: String
     author: String
     url: String
+    user: String
```

Comment:
I don't think we'll expose this to the frontend, so I'm unclear if it's useful to have it here.  cc @c3ho for thoughts.

Reply:
I'm unsure but if a user is logged in and wants to delete a post, won't this need to be exposed on the frontend somehow.

Reply:
The server always knows what the current authenticated user's `id` is, since it's stored in the session.  We don't need to round-trip it through the frontend.

Reply:
Ah - I can't envision what you mean, but that can be a road crossed when I'm writing the code for that part :D.

Reply:
What @humphd said. I don't imagine us having to use user in the front end as each post displays the author's real name and if we need to uniquely identify two authors we might have the same name + last name we can always use a combination of the feedId + author.

Reply:
Fixed, based on @c3ho and @humphd points..